website/cue/reference/components/sinks/loki.cue

```diff
@@ -105,26 +105,43 @@ components: sinks: loki: {
 		label_expansion: {
 			title: "Label Expansion"
 			body: """
-				The `labels` option can be passed keys suffixed with `*` to
-				allow for setting multiple keys based on the contents of an
-				object. For example, with an object:
+				The `labels` option can be passed keys with `*` or prefixes ending with `*` to
+				allow for setting multiple keys based on the contents of an object. Static keys
+				override dynamically defined keys. For example, with an object:
 
 				```json
-				{"kubernetes":{"pod_labels":{"app":"web-server","name":"unicorn"}}}
+				{
+					"kubernetes": {
+						"pod_labels": {
+							"app": "web-server",
+							"name": "unicorn"
+						}
+					},
+					"metadata": {
+						"cluster_name": "operations",
+						"cluster_environment": "development",
+						"cluster_version": "1.2.3"
+					}
+				}
 				```
 
 				and a configuration:
 
 				```toml
 				[sinks.my_sink_id.labels]
-				pod_labels_*: "{{ kubernetes.pod_labels }}"
+				\"pod_labels_*\": "{{ kubernetes.pod_labels }}"
+				\"*\": "{{ metadata }}"
```

Comment:
Ah, is this invalid yaml without the `*`'s being quoted?

Reply:
Good question. My IDE highlighted it as invalid, but I didn't check if this would really be invalid/unparseable.
Quoted worked and I had no complaints from the editor.

As per the toml 1.0 specs it is invalid: https://toml.io/en/v1.0.0#keys
> Bare keys may only contain ASCII letters, ASCII digits, underscores, and dashes (A-Za-z0-9_-). Note that bare keys are allowed to be composed of only ASCII digits, e.g. 1234, but are always interpreted as strings.

> Quoted keys follow the exact same rules as either basic strings or literal strings and allow you to use a much broader set of key names. Best practice is to use bare keys except when absolutely necessary.